src/applications/facility-locator/reducers/searchResult.js

```diff
@@ -27,9 +27,10 @@ export const SearchResultReducer = (state = INITIAL_STATE, action) => {
       };
     case SEARCH_FAILED:
       if (action.error) {
-        // Log error with Sentry
-        // TODO: Figure out why this is causing errors so we can capture correctly
-        // captureMessage(`Locator Search Failed: ${action.error}`);
```

Comment:
Does removing this mean the error conditions are adequately understood now?

Reply:
The only errors I've seen for this particular call recently are the timeout errors, which are currently being handled properly.